modules/backend/widgets/Filter.php

```diff
@@ -1,11 +1,14 @@
 <?php namespace Backend\Widgets;
 
+use Backend;
+use Backend\Classes\FormField;
+use Backend\FormWidgets\DatePicker;
+use Carbon\Carbon;
 use Db;
 use Str;
 use Lang;
-use Backend;
```

Comment:
Don't move stuff around just because you feel like it. October isn't super consistent with the order of it's use statements but it usually goes by length of the `use X` statement.